# We are starting with the algorithim workbench questions.
## Question 1 algorithim workbench
Write a  loop that lets the user enter a number. The number should be multiplied by 10, and the result assigned to a variable named . The loop should iterate as long as  is less than 100

In [ ]:
result = 0

while result < 100:


## First you set the varible and then you also set the loop up for it being less then 100
Then from there you can set up the rest of the program

In [1]:
result = 0


while result < 100:

    number = float(input("Enter a number: "))
    
    result = number * 10
    
    print(f"The result is {result}")


Enter a number: 50
The result is 500.0


### My prediction is that it will give us the right number and give us the number mutiplied by 10 as long as it is not above 100

## Question 3 Algorithim workbench
Write a  loop that displays the following set of numbers: 
0, 10, 20, 30, 40, 50.....1000

## This code is very simple since we are using a FOR loop and the loop tells you what its gonna do.  

In [2]:
for number in range(0, 1001, 10):
    print(number)

0
10
20
30
40
50
60
70
80
90
100
110
120
130
140
150
160
170
180
190
200
210
220
230
240
250
260
270
280
290
300
310
320
330
340
350
360
370
380
390
400
410
420
430
440
450
460
470
480
490
500
510
520
530
540
550
560
570
580
590
600
610
620
630
640
650
660
670
680
690
700
710
720
730
740
750
760
770
780
790
800
810
820
830
840
850
860
870
880
890
900
910
920
930
940
950
960
970
980
990
1000


# Programming exercises, questions: 1, 4

## Question 1: The Bug collecter
A bug collector collects bugs every day for five days. Write a program that keeps a running total of the number of bugs collected during the five days. The loop should ask for the number of bugs collected for each day, and when the loop is finished, the program should display the total number of bugs collected.

### First you initialize the the total bugs and write a for loop that will go for 5 days.

In [3]:
total_bugs = 0

for day in range(1, 6):

SyntaxError: incomplete input (2896573732.py, line 3)

### Once thats written you can write the rest of the filler code to fill the boundrys out more.

In [4]:
total_bugs = 0

for day in range(1, 6):

    bugs_collected = int(input(f"Enter the number of bugs collected on day {day}: "))
    
    total_bugs += bugs_collected

print(f"The total number of bugs collected over 5 days is: {total_bugs}")


Enter the number of bugs collected on day 1: 20
Enter the number of bugs collected on day 2: 5
Enter the number of bugs collected on day 3: 15
Enter the number of bugs collected on day 4: 24
Enter the number of bugs collected on day 5: 30
The total number of bugs collected over 5 days is: 94


### My prediction is that it will ask for you to enter a number of bugs and then will tell you the total collected at the end of 5 days worth of bugs 

# Question 4: Distance Travelled
The distance a vehicle travels can be calculated as follows:distance = speed × timeFor example, if a train travels 40 miles per hour for three hours, the distance traveled is 120 miles. Write a program that asks the user for the speed of a vehicle (in miles per hour) and the number of hours it has traveled. It should then use a loop to display the distance the vehicle has traveled for each hour of that time period. |

## Start by writing the print statments that are the questions it will ask and then also initalize any variables you have. 

In [5]:
speed = float(input("Enter the speed of the vehicle (in miles per hour): "))

hours = int(input("Enter the number of hours the vehicle has traveled: "))

print("\nHour\tDistance Traveled")
print("-------------------------")


Enter the speed of the vehicle (in miles per hour): 60
Enter the number of hours the vehicle has traveled: 4

Hour	Distance Traveled
-------------------------


## From there you will want to write the equation and any other small things you will need. 

In [6]:
speed = float(input("Enter the speed of the vehicle (in miles per hour): "))

hours = int(input("Enter the number of hours the vehicle has traveled: "))

print("\nHour\tDistance Traveled")
print("-------------------------")

for hour in range(1, hours + 1):

    distance = speed * hour
    
    print(f"{hour}\t{distance:.2f}")

Enter the speed of the vehicle (in miles per hour): 60
Enter the number of hours the vehicle has traveled: 5

Hour	Distance Traveled
-------------------------
1	60.00
2	120.00
3	180.00
4	240.00
5	300.00


### My prediction is that the question will ask how fast I am going and then from there caluculate using my equation to see how many miles I can travel in that amount of hours. 

# Question 11: Weight Loss
 a moderately active person cuts their calorie intake by 500 calories a day, they can typically lose about 4 pounds a month. Write a program that lets the user enter their starting weight, then creates and displays a table showing what their expected weight will be at the end of each month for the next 6 months if they stay on this diet.

## First you want to initialize anything taht is needed and malke the proper print statments 

In [7]:
starting_weight = float(input("Enter your starting weight (in pounds): "))

weight_loss_per_month = 4

print("\nMonth\tExpected Weight")
print("-------------------------")

Enter your starting weight (in pounds): 155

Month	Expected Weight
-------------------------


## Next you wnat to write a for statment and any other small print statments to enclose the boundry of this problem

In [8]:
starting_weight = float(input("Enter your starting weight (in pounds): "))

weight_loss_per_month = 4

print("\nMonth\tExpected Weight")
print("-------------------------")

for month in range(1, 7):

    expected_weight = starting_weight - (weight_loss_per_month * month)
    

    print(f"{month}\t{expected_weight:.2f}")



Enter your starting weight (in pounds): 250

Month	Expected Weight
-------------------------
1	246.00
2	242.00
3	238.00
4	234.00
5	230.00
6	226.00


### My prediction is that the code will aks what my starting weight is and then calulate if I were to loose 4 pounds and checks in in 6 months 

# Question 14: Write a program that uses nested loops to make this pattern 
    ##
    #  #
    #    #
    #      #
    #        #
    #          #

## Start with everything normal with initilizing varibles and printing the statments. But you will have to make nested loops for this one so lets go ahead and do that too 

In [9]:
num_lines = 6

for i in range(num_lines):

    print('#', end='')
    

    if i > 0:
        for j in range(i * 2):
            print(' ', end='')
    
    print('#')


##
#  #
#    #
#      #
#        #
#          #


### As predicted It printed the right pattern that I wanted it to create!!!